mesonbuild/interpreterbase/decorators.py

```diff
@@ -782,6 +782,64 @@ def log_usage_warning(self, tv: str, location: T.Optional['mparser.BaseNode']) -
         mlog.deprecation(*args, location=location)
 
 
+class FeatureExperimental(FeatureCheckBase):
+
+    """Checks for experimental features"""
+
+    feature_registry = {}
```

Comment:
```suggestion
    feature_registry = {}
    unconditional = True
```

I added this for FeatureBroken. It means that we are using the FeatureCheck framework for consistency and reporting, but do not actually want to treat it like normal "features" and only emit messages to people that have opted into them by specifying which `meson_version`s they have a minimum requirement on.